ios/FluentUI/Notification/FluentNotification.swift

```diff
@@ -67,6 +71,7 @@ public struct FluentNotification: View, ConfigurableTokenizedControl {
                 title: String? = nil,
                 attributedTitle: NSAttributedString? = nil,
                 image: UIImage? = nil,
+                trailingImage: UIImage? = nil,
```

Comment:
Mostly for my knowledge, but does it matter where you add new optional params? Should this be added at the end of the list? It would need to be in other languages, like C#, to keep this from being a breaking change.

Reply:
I think that's a reasonable concern, but I think this should be fine in this case. Since it's in Swift, and not available in Objective C, you can just leave out arguments. If this were available in Objective C, I would've needed to add a convenience init.